### PR TITLE
Upgrade rubocop to fix Style/WordArray warning.

### PR DESCRIPTION
**Pull request by GordonDiggs**
A few users have run into this problem, and upgrading rubocop
(specifically the parsing dependencies) fixes the issue. Example engine
output is shown below.

Before this change
==================

```
An error occurred while Style/WordArray cop was inspecting
config/initializers/logs.rb.
To see the complete backtrace run rubocop -d.
```

After this change
=================

```
warning: parser/current is loading parser/ruby22, which recognizes
warning: 2.2.3-compliant syntax, but you are running 2.2.2.
warning: please see
https://github.com/whitequark/parser#compatibility-with-ruby-mri.
```

Note: this version warning is new, but as there were no syntax changes
introduced in ruby 2.2.3, I don't believe it is an issue.
(https://www.ruby-lang.org/en/news/2015/08/18/ruby-2-2-3-released/)

@codeclimate/review

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -61,3 +63,6 @@ DEPENDENCIES
   rubocop
   rubocop-rspec
   safe_yaml
+
+BUNDLED WITH
+   1.10.6
```
Comment (ABaldwinHunter) [inline Gemfile.lock]: Is the `Bundled with` addition intentional or impactful?

**Comment by GordonDiggs** [comment]
@ABaldwinHunter does this interfere with your points work?

File: Gemfile.lock (hunk 1 above)
Comment (GordonDiggs) [inline Gemfile.lock]: Neither. Will remove though

**Comment by ABaldwinHunter** [comment]
@GordonDiggs I don't think it interferes.

Minor comment but LGTM

**Comment by ABaldwinHunter** [comment]
Also, the points work is already merged

**Comment by ABaldwinHunter** [comment]
Thanks for heads up

**Pull request merged**